cocos/3d/CCTerrain.h

```diff
@@ -424,7 +424,7 @@ class CC_DLL Terrain : public Node
     /**
      * get the terrain's size
      */
-    Size getTerrainSize() const { return Size(_imageWidth, _imageHeight); }
+    Size getTerrainSize() const { return Size(static_cast<float>(_imageWidth), static_cast<float>(_imageHeight)); }
```

Comment:
@minggo the code, internally, treat it as int. in this particular case, it seems better to cast it to float in `getTerrainSize`